pkg/driver/libsql/libsql.go

```diff
@@ -0,0 +1,246 @@
+//go:build cgo
+// +build cgo
+
+package libsql
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"io"
+	"net/url"
+	"strings"
+
+	"github.com/amacneil/dbmate/v2/pkg/dbmate"
+	"github.com/amacneil/dbmate/v2/pkg/dbutil"
+
+	"github.com/lib/pq"
+	_ "github.com/libsql/libsql-client-go/libsql" // database/sql driver
+)
+
+func init() {
+	dbmate.RegisterDriver(NewDriver, "libsql")
+	dbmate.RegisterDriver(NewDriver, "http")
+	dbmate.RegisterDriver(NewDriver, "https")
+	dbmate.RegisterDriver(NewDriver, "ws")
+	dbmate.RegisterDriver(NewDriver, "wss")
```

Comment:
huh? why is it accessible over http + websocket?

Reply:
Check this out 🙂

https://github.com/libsql/sqld

Reply:
Registering the very generic `http`, `https`, `ws` and `wss` to this driver is probably going to bite us in the future.  I would rather see us register `libsql+http`, `libsql+https`, `libsql+ws` and `libsql+wss` and peel off the `libsql+` prefix inside `ConnectionString()`.